.github/workflows/wheels.yml

```diff
@@ -0,0 +1,53 @@
+name: RMM wheels
+
+on:
+  push:
+    branches:
+      - 'pull-request/[0-9]+'
+  workflow_call:
+    inputs:
+      versioneer-override:
+        type: string
+        default: ''
+      build-tag:
+        type: string
+        default: ''
+      branch:
+        required: true
+        type: string
+      date:
+        required: true
+        type: string
+      sha:
+        required: true
+        type: string
+      build-type:
+        type: string
+        default: nightly
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  rmm-wheels:
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux.yml@main
+    with:
+      repo: rapidsai/rmm
+
+      build-type: ${{ inputs.build-type || 'pull-request' }}
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+
+      package-dir: python
+      package-name: "rmm"
+
+      python-package-versioneer-override: ${{ inputs.versioneer-override }}
+      python-package-build-tag: ${{ inputs.build-tag }}
+
+      skbuild-configure-options: "-DRMM_BUILD_WHEELS=ON"
+
+      test-extras: "test"
+      test-unittest: "python -m pytest -v ./python/rmm/tests"
```

Comment:
If pytest-xdist is in the test deps, then use it to parallelize.

Reply:
We never parallelize rmm tests because many of them test allocations that would OOM if done in parallel. You can see that [we aren't parallelizing in the current CI either](https://github.com/rapidsai/rmm/blob/branch-22.12/ci/test_python.sh#L39), and that goes back to [the old gpuCI scripts](https://github.com/rapidsai/rmm/blob/branch-22.08/ci/gpu/build.sh#L125). That does probably mean that we can drop pytest-xdist from the requirements though, I hadn't really noticed it there. Will remove.